YourssuCalculator/Presentation/Home/HomeViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  ViewController.swift
+//  YourssuCalculator
+//
+//  Created by 홍준혁 on 2023/04/26.
+//
+
+import UIKit
+
+import SnapKit
+
+class HomeViewController: UIViewController {
+    
+    private let homeView = HomeView()
+    private let homeViewModel: HomeViewModel?
```

Comment:
```suggestion
    private var homeViewModel: HomeViewModelInputOutput?
```
이미 다 만들어두시고 나서 이거 적용을 까먹으신 것 같네요. 추상화 좋아요^~^

Reply:
[0760f44](https://github.com/hongjunehuke/Yourssu-iOS-Incubate/pull/2/commits/0760f441d1c07aecd291323d8aeae008038a6c97)